### PR TITLE
[RTA] Adds RTAs to Windows Rules - 2

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Adds RTA coverage to the following (51) rules:

- Microsoft Exchange Worker Spawning Suspicious Processes
<details>
<summary>Screenshot</summary>

image

</details>

- Web Shell Detection: Script Process Child of Common Web Processes
<details>
<summary>Screenshot</summary>

image

</details>

- Remote File Copy to a Hidden Share
<details>
<summary>Screenshot</summary>

image

</details>

- High Number of Process and/or Service Terminations
<details>
<summary>Screenshot</summary>

image

</details>

- Mounting Hidden or WebDav Remote Shares
<details>
<summary>Screenshot</summary>

image

</details>

- New ActiveSyncAllowedDeviceID Added via PowerShell
<details>
<summary>Screenshot</summary>

image

</details>

- Persistence via WMI Event Subscription
<details>
<summary>Screenshot</summary>

image

</details>

- Microsoft Exchange Server UM Writing Suspicious Files
<details>
<summary>Screenshot</summary>

image

</details>

- Persistence via Microsoft Office AddIns
<details>
<summary>Screenshot</summary>

image

</details>

- Lateral Movement via Startup Folder
<details>
<summary>Screenshot</summary>

image

</details>

- Persistence via Microsoft Outlook VBA
<details>
<summary>Screenshot</summary>

image

</details>

- Persistence via PowerShell profile
<details>
<summary>Screenshot</summary>

image

</details>

- Startup Persistence by a Suspicious Process
<details>
<summary>Screenshot</summary>

image

</details>

- Persistence via Scheduled Job Creation
<details>
<summary>Screenshot</summary>

image

</details>

- Persistent Scripts in the Startup Directory
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious Print Spooler File Deletion
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious DLL Loaded for Persistence or Privilege Escalation
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious RDP ActiveX Client Loaded
<details>
<summary>Screenshot</summary>

image

</details>

- Windows Script Interpreter Executing Process via WMI
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious Image Load (taskschd.dll) from MS Office
<details>
<summary>Screenshot</summary>

image

</details>

- Volume Shadow Copy Deletion via PowerShell
<details>
<summary>Screenshot</summary>

image

</details>

- Unusual Persistence via Services Registry
<details>
<summary>Screenshot</summary>

image

</details>

- Registry Persistence via AppCert DLL
<details>
<summary>Screenshot</summary>

image

</details>

- Registry Persistence via AppInit DLL
<details>
<summary>Screenshot</summary>

image

</details>

- Disabling User Account Control via Registry Modification
<details>
<summary>Screenshot</summary>

image

</details>

- Installation of Security Support Provider
<details>
<summary>Screenshot</summary>

image

</details>

- Potential LSA Authentication Package Abuse
<details>
<summary>Screenshot</summary>

image

</details>

- NullSessionPipe Registry Modification
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious Print Spooler Point and Print DLL
<details>
<summary>Screenshot</summary>

image

</details>

- Potential Port Monitor or Print Processor Registration Abuse
<details>
<summary>Screenshot</summary>

image

</details>

- Potential Remote Desktop Shadowing Activity
<details>
<summary>Screenshot</summary>

image

</details>

- Installation of Custom Shim Databases
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious Startup Shell Folder Modification
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious ImagePath Service Creation
<details>
<summary>Screenshot</summary>

image

</details>

- Potential Persistence via Time Provider Modification
<details>
<summary>Screenshot</summary>

image

</details>

- Privilege Escalation via Windir Environment Variable
<details>
<summary>Screenshot</summary>

image

</details>

- RDP Enabled via Registry
<details>
<summary>Screenshot</summary>

image

</details>

- Persistence via BITS Job Notify Cmdline
<details>
<summary>Screenshot</summary>

image

</details>

- Unusual Service Host Child Process - Childless Service
<details>
<summary>Screenshot</summary>

image

</details>

- Persistence via TelemetryController Scheduled Task Hijack
<details>
<summary>Screenshot</summary>

image

</details>

- Unusual Child Process of dns.exe
<details>
<summary>Screenshot</summary>

image

</details>

- Unusual File Modification by dns.exe
<details>
<summary>Screenshot</summary>

image

</details>

- Microsoft Exchange Server UM Spawning Suspicious Processes
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious Explorer Child Process
<details>
<summary>Screenshot</summary>

image

</details>

- System Shells via Services
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious Execution via Scheduled Task
<details>
<summary>Screenshot</summary>

image

</details>

- Execution of Persistent Suspicious Program
<details>
<summary>Screenshot</summary>

image

</details>

- Suspicious MS Outlook Child Process
<details>
<summary>Screenshot</summary>

image

</details>

- Service Control Spawned via Script Interpreter
<details>
<summary>Screenshot</summary>

image

</details>

- UAC Bypass Attempt with IEditionUpgradeManager Elevated COM Interface
<details>
<summary>Screenshot</summary>

image

</details>

- UAC Bypass via Windows Firewall Snap-In Hijack
<details>
<summary>Screenshot</summary>

image

</details>

Also adjusts the metadata of some RTAs to pass flake8 tests, simplify the uncommon persistence one

### CODE REVIEW

File: rta/at_command.py [hunk 1]
```diff
@@ -17,7 +17,13 @@
 from . import RtaMetadata
 
 
-metadata = RtaMetadata(uuid="961d7a1f-7bad-41d5-a3d9-8e8a2f59a824", platforms=["windows"], endpoint=[], siem=[], techniques=[])
+metadata = RtaMetadata(
+    uuid="961d7a1f-7bad-41d5-a3d9-8e8a2f59a824",
+    platforms=["windows"],
+    endpoint=[],
+    siem=[],
```
Comment (terrancedejesus) [inline rta/at_command.py:24]: Do we have any rules to reference for this?

**Review by terrancedejesus — APPROVED** [review]
Approving, great work! A lot in this PR. I would suggest that we add the SIEM, Endpoint rules and Techniques wherever valid.

File: rta/at_command.py (hunk 1 above)
Comment (w0rk3r) [reply rta/at_command.py:24]: This is just a linting modification, audit/adjusts to mappins on existing RTAs will be done in https://github.com/elastic/ia-trade-team/issues/62

**Review by eric-forte-elastic — APPROVED** [review]
LGTM!

**Pull request merged**